intro.md

```diff
@@ -1,17 +1,26 @@
 ### What does it do?
 
-- xxx
-- xxx
-- xxx
+- Provides an architecture that enables PRO AV User Stories in an NMOS environment
```

Comment:
Since the user stories are all focused on replacing more tradition cabling perhaps that would be mentioned here?

```suggestion
- Provides an architecture that enables PRO AV User Stories, previously handled by mediums like HDMI or DisplayPort, in an NMOS environment.
```

Reply:
Looks good. Will add